package.json

```diff
@@ -132,7 +137,8 @@
             "explorer": [
                 {
                     "id": "xmlTreeView",
-                    "name": "XML Document"
+                    "name": "XML Document",
+                    "when": "config.xmlTools.treeViewEnabled != false"
```

Comment:
Should be:

resourceLangId == 'xml' && config.xmlTools.treeViewEnabled != false